src/resources/lib/modules/bluetooth.py

```diff
@@ -1126,8 +1126,9 @@ def stop(self):
 
     def run(self):
         try:
+            xbmcm = xbmc.Monitor()
             self.oe.dbg_log('bluetooth::discoveryThread::run', 'enter_function', 0)
-            while not self.stopped and not xbmc.abortRequested:
+            while not self.stopped and not xbmcm.abortRequested():
                 current_time = time.time()
                 if current_time > self.last_run + 5:
                     if self.main_menu.getSelectedItem().getProperty('modul') != 'bluetooth' or not hasattr(self.oe.dictModules['bluetooth'], 'discovery_thread'):
```

Comment:
Always better to use xbmcm.waitForAbort(1) instead of time.sleep(1)

Reply:
Done